lib/beeswaxapi/request.rb

```diff
@@ -118,6 +118,8 @@ def after_request_log(target_url, opts, response)
 
     def parsed_body(body)
       Yajl.load(body, symbolize_keys: true)
+    rescue Yajl::ParseError
+      {body: body, success: false}
```

Comment:
Can we keep the rescue but return the original body in case of ParseError?, something like : 
```
 def parsed_body(body)
    Yajl.load(body, symbolize_keys: true)[:results]
 rescue Yajl::ParseError
    body
  end
```
In that context, the `parsed_body` method only parses the response body, but it doesn't set the success key value. It looks like a general method and is also used in the **sucess_response_handler** method.
And then, L137 could be rewritten like:
`@response = Response.new(success: true, code: code, payload: parsed_body(body)) `

And L141: 
`@response = Response.new(success: false, code: code, body: parsed_body(body)) `
WDYT?

Reply:
BS usually provides a response with the next schema:
```json
{
    "success": true,
    "payload": {
            "user_id": 1,
            "first_name": "Danny"
            ...
    },
  "errors": {}
```
that is why we shouldn't explicitly set success for a happy path.

Reply:
yeah, because we expect to have a particular data structure from them.